prdoc/pr_4976.prdoc

```diff
@@ -0,0 +1,15 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Add pub to xcm::v4::PalletInfo
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Forgot to make v4 PalletInfo fields public. Without them we cannot make use of the struct.
+
+crates:
+  - name: staging-xcm
+    bump: patch
+    validate: false
```

Comment:
```suggestion
```

I hope that we fixed the semver check in the meantime. Can you maybe try to see if the CI goes green without this line?

Reply:
One question I had is whether or not the prdoc needs to be backported as well

Reply:
Still updated it though